libvast/src/system/importer.cpp

```diff
@@ -44,79 +44,41 @@ importer_state::~importer_state() {
 }
 
 caf::error importer_state::read_state() {
-  VAST_TRACE("");
-  id_generators.clear();
-  auto file = dir / "available_ids";
+  auto file = dir / "next_id";
   if (exists(file)) {
-    VAST_DEBUG(self, "reads persistent state from", to_string(file));
+    VAST_VERBOSE(self, "reads persistent state from", to_string(file));
```

Comment:
```suggestion
    VAST_VERBOSE(self, "reads persistent state from", file);
```

I have not tested this locally, but `to_string` should already be called implicitly here. Same goes for the other two log messages in this function.